tools/check_go_mod.sh

```diff
@@ -14,8 +14,7 @@ function run() {
         git diff -- go.mod go.sum
 
         echo -e "\nThis means that Go module definition is not consistent."
-        echo "Please run 'go mod tidy' and commit changes."
-        die
+        die "Please run 'go mod tidy' and commit changes."
```

Comment:
Similar here, will running `go mod tidy` again fix anything?

Reply:
The same case as in https://github.com/kentik/terraform-provider-kentik-synthetics/pull/69#discussion_r765002422